src/PKSim.Core/Services/SimulationBuildingBlockUpdater.cs

```diff
@@ -140,7 +142,33 @@ public bool QuickUpdatePossibleFor(IPKSimBuildingBlock templateBuildingBlock, Us
          if (!BuildingBlockSupportsQuickUpdate(templateBuildingBlock))
             return false;
 
-         return templateBuildingBlock.StructureVersion == usedBuildingBlock.StructureVersion;
+         //not the same structure, easy return
+         var sameStructureVersion = templateBuildingBlock.StructureVersion == usedBuildingBlock.StructureVersion;
+         if (!sameStructureVersion)
+            return false;
+
+         //For individual, there is a special handling required as we need to also check that the used expression profile have hte same structure version
+         if (templateBuildingBlock is not Individual individualTemplate)
+            return true;
+
+         //It has to be available by construction
+         var usedIndividual = usedBuildingBlock.BuildingBlock as Individual;
+         //but we return false just in case :)
+         if (usedIndividual == null)
+            return false;
+
+         //let's compare the expression profile in each individuals and see if they are comparable
+         //not the same length of expression profile, structural change
+         if (usedIndividual.AllExpressionProfiles().Count != individualTemplate.AllExpressionProfiles().Count)
+            return false;
```

Comment:
Is it actually possible, that `sameStructureVersion` in the code above is true, but we have a different number of expression profiles?

Reply:
I don't think so. you are right this check can be removed.